src/components/Booklisting/Booklisting.jsx

```diff
@@ -0,0 +1,21 @@
+import "./Booklisting.css"
+import { useNavigate } from 'react-router-dom'
+
+// booklisting will take props to display JSON data from Home 
+function BookListing({title, author, image, genre, id,}){
+    const navigate = useNavigate();
+
+    function bookClick() {
+        navigate('/books/' + id)
+    }
+
+    return(
+        <div className="singleBookWrapper" onClick={bookClick}>

```

Comment:
You don't have to change anything, just sharing for information. The `onClick` approach here introduces an accessibility issue. The list of books cannot be navigated by a keyboard and screen reader users will not be informed that the book can be clicked on.